src/components/study-container.js

```diff
@@ -304,6 +306,35 @@ export function StudyContainer({ view, onChangeTab }) {
         }
     }, [studyUuid, loadTree]);
 
+    useEffect(() => {
+        const appName = 'GridStudy';
+
+        if (studyUuid) {
+            fetchElementAndParentsInfo(studyUuid)
+                .then((response) => {
+                    const study = response[0];
+                    const parents = response
+                        .slice(1)
+                        .map((parent) => parent.elementName);
+
+                    document.title = computePageTitle(appName, study, parents);
+                })
+                .catch((errorMessage) => {
+                    document.title = appName;
```

Comment:
remove

Reply:
not removed to be able to come back to the default title if the study's uuid is changed and something wrong happens when loading it's name/path

Reply:
ok